addons/spreadsheet/static/src/chart/plugins/odoo_chart_ui_plugin.js

```diff
@@ -60,9 +60,17 @@ export default class OdooChartUIPlugin extends UIPlugin {
                 switch (cmd.definition.type) {
                     case "odoo_pie":
                     case "odoo_bar":
-                    case "odoo_line":
+                    case "odoo_line": {
+                        const dataSource = this.getChartDataSource(cmd.id);
+                        if (
+                            new Domain(dataSource.getComputedDomain()).toString() !==
```

Comment:
It shouldn't compare to the computed domain, right ?

Reply:
In fact getComputedDomain returns this._customDomain, which seems to be the actual domain no ?

Reply:
`this._customDomain` includes the domain parts of global filters (if there is one which applies)